.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: python
+python: 2.7
+
+sudo: false
+
+# Pre-install packages for the ubuntu distribution
+cache:
+    apt: true
+    directories:
+        - $HOME/.cache/pip
+
+addons:
+    apt:
+    packages:
+        - libatlas3gf-base
+        - libatlas-dev
+        - python-scipy
+
+# command to install dependencies
+install:
+    - pip install -U pip wheel
+    - pip install -r requirements.txt
+    - python setup.py develop
```

Comment:
You probably either want to just run `setup.py` here or change to the inverse_covariance/pyquic directory and run `make`